bot/zconfig.py

```diff
@@ -0,0 +1,181 @@
+#
```

Comment:
This file is not necessary, right?

Reply:
This should be added to `config.py.zephyr.sample`, as it indicates with what config these tests should run. I corrected it now